crates/bevy_ecs/src/change_detection.rs

```diff
@@ -197,3 +197,4 @@ pub struct ReflectMut<'a> {
 
 #[cfg(feature = "bevy_reflect")]
 change_detection_impl!(ReflectMut<'a>, dyn Reflect,);
+impl_into_inner!(ReflectMut<'a>, dyn Reflect,);
```

Comment:
I would have expected this to need a similar conditional compilation to the item above.

Reply:
added it